cmd/kubeadm/app/cmd/init.go

```diff
@@ -443,18 +445,106 @@ func (i *Init) Run(out io.Writer) error {
 	}
 
 	ctx := map[string]string{
-		"KubeConfigPath": adminKubeConfigPath,
 		"Token":          i.cfg.Token,
 		"CAPubKeyPin":    pubkeypin.Hash(caCert),
 		"MasterHostPort": masterHostPort,
 	}
+
+	kubeConfigInfo := fmt.Sprintf(`
+To start using your cluster, you need to run the following as a regular user:
+
+	  mkdir -p $HOME/.kube
+	  sudo cp -i %s $HOME/.kube/config
+	  sudo chown $(id -u):$(id -g) $HOME/.kube/config
+
+`, adminKubeConfigPath)
+
 	if i.skipTokenPrint {
 		ctx["Token"] = "<value withheld>"
 	}
 
+	if i.defaultUser == "" {
+		ctx["KubeConfigInfo"] = kubeConfigInfo
+	} else {
+		ctx["KubeConfigInfo"] = ""
+		if err := copyConfigForDefaultUser(i.defaultUser, adminKubeConfigPath); err != nil {
+			return fmt.Errorf("error copying configuration for user %q: %v", i.defaultUser, err)
```

Comment:
Shouldn't we still have the previous instructions for a failure?

Reply:
seems like the correct thing to do. i will amend the commit.